pkg/csi/service/common/vsphereutil.go

```diff
@@ -437,8 +437,8 @@ func getDatastoreMoRefs(datastores []*vsphere.DatastoreInfo) []vim25types.Manage
 	return datastoreMoRefs
 }
 
-// Helper function to get DatastoreMoRef for given datastoreURL in the given virtual center.
-func getDatastore(ctx context.Context, vc *vsphere.VirtualCenter, datastoreURL string) (vim25types.ManagedObjectReference, error) {
+// GetDatastore is a helper function to get DatastoreMoRef for given datastoreURL in the given virtual center.
+func GetDatastore(ctx context.Context, vc *vsphere.VirtualCenter, datastoreURL string) (vim25types.ManagedObjectReference, error) {
```

Comment:
This function is not used. I think there is an bug in this function, when I called this function with the URL it returned error saying "failed to fetch datacenter given dcPath", this is because we store in VirtualCenterConfig.DatacenterPaths the moids of datacenters which is not what govmomi.finder expects.